packages/orbit-components/src/Inline/index.js

```diff
@@ -0,0 +1,56 @@
+// @flow
+import * as React from "react";
+import styled, { css } from "styled-components";
+
+import defaultTheme from "../defaultTheme";
+import { DEVICES as DEVICES_CONSTS } from "../utils/mediaQuery/consts";
+import media from "../utils/mediaQuery";
+import { normalize } from "./helpers";
+
+import type { Props } from "./index";
+
+const StyledInline = styled(({ as: Element, children }) => (
+  <Element>{children}</Element>
+)).attrs(({ dataTest }) => ({ "data-test": dataTest }))`
```

Comment:
Would define the `data-test` on the return inside `styled` factory. It seems redundant to call another function.